### PR TITLE
[API] Add Sharing Type w/ Message/Title/URL

### LINKED ISSUE #206: Add URL and Message/Description Items to Graph.
## User Story

As a developer I want to have URL and message/description data passed in through the Graph so that I can share the most accurate information.

#### Problem Details _(Optional)_
Right now we only have the title of a content item being passed down through the graph. For sharing, we also need at least one of a URL or Message/Description. Ideally both.

#### Acceptance Criteria _(Optional)_

* [ ] It should pass down URL and message/description data for content items.

### PR DESCRIPTION
**Pull request by vinnyjth**
Fixes #206

Adds the ability to query 
```
sharing {
  url
  message
  title
}
```
on a content item, with support for additional types easy to add as needed.

Also in this PR, fixed warnings when running tests on API. 

## Creator

- [x] Build relevant tests if any apply
- [x] Ensure there are no new warnings (in app AND in storybook)
- [x] Upload an animated GIF showcasing the solution (if it applies)
- [x] Set a relevant reviewer
- [x] PR has a relevant title that adheres to our patch-notes/change-log style

## Reviewer

- [ ] Review CI build results
- [ ] Review function and form on iOS
- [ ] Review function and form on Android
- [ ] Review new stories if any apply
- [ ] Review test coverage: Are all new or changed states tested?
- [ ] Review new test logic
- [ ] Review the change-log entry
- [ ] Review that the PR title adheres to our patch-notes/change-log style

### CODE REVIEW

**Comment by richarddubay** [comment]
@vinnyjth Also, it looks like test coverage went down? Anything we can do about that?

File: packages/apolloschurchapp/package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "android": "react-native run-android",
     "codecov": "cat ./coverage/lcov.info | codecov",
     "fixlint": "eslint ./src --fix",
-    "ios": "react-native run-ios --simulator=\"iPhone X\"",
+    "ios": "react-native run-ios --simulator=\"iPhone 7\"",
```
[the remark this answers is not in the document]
Comment (vinnyjth) [reply packages/apolloschurchapp/package.json]: My bad :)

**Comment by vinnyjth** [comment]
@richarddubay Pushed up a change that should increase test coverage above where it was before this PR :)

**Comment by vinnyjth** [comment]
@richarddubay Hmmm - didn't do as much as I could. I don't think there's an easy way to bump the coverage back up b/c we don't use the `Media` or `MediaSource` types directly.

**Pull request merged**